mm2src/database/stats_swaps.rs

```diff
@@ -243,6 +251,51 @@ fn insert_stats_taker_swap_sql_init(swap: &TakerSavedSwap) -> Option<(&'static s
     Some((INSERT_STATS_SWAP_ON_INIT, params))
 }
 
+fn update_stats_swap_db(
+    ctx: &MmArc,
+    maker_coin_usd_price: &str,
+    taker_coin_usd_price: &str,
+    uuid: &str,
+) -> SqlResult<()> {
+    debug!("Updating swap coins usd price for {} to the SQLite database", uuid);
+    let conn = ctx.sqlite_connection();
+    let params = [uuid, maker_coin_usd_price, taker_coin_usd_price];
+
+    conn.execute(UPDATE_STATS_SWAP, &params).map(|_| ())
+}
+
+pub async fn update_stats_swap_coins_price_to_db(
+    ctx: &MmArc,
+    swap: &SavedSwap,
+    base_usd_price: BigDecimal,
+    rel_usd_price: BigDecimal,
+) -> Result<(), String> {
+    match &swap {
```

Comment:
You can avoid duplicating the code just by using [SavedSwap::uuid](https://github.com/KomodoPlatform/atomicDEX-API/blob/dev/mm2src/lp_swap/saved_swap.rs#L59-L64).